docs/modules/ROOT/pages/erc4626.adoc

```diff
@@ -157,28 +164,30 @@ For the attacker to dilute that deposit to 0 shares, causing the user to lose al
 
 [stem]
 ++++
-10^\delta \times u \times \frac{1+a_0}{1+a_0+a_1} < 1
+10^\delta \times u \leq \frac{a_1}{1 + a_0}
```

Comment:
Here and on line 172, there is a change from `<` to `\leq` ... but then on line 177 and 182 it is `\le`

Reply:
Changed all to leq